game_eggs/steamcmd_servers/stationeers/stationeers_vanilla/egg-stationeers.json

```diff
@@ -4,50 +4,68 @@
         "version": "PTDL_v2",
         "update_url": null
     },
-    "exported_at": "2022-12-30T11:39:51+01:00",
-    "name": "Stationeers",
-    "author": "eggs@goover.dev",
-    "description": "Stationeers Server",
-    "features": [
-        "steam_disk_space"
-    ],
+    "exported_at": "2024-02-21T04:35:19+01:00",
+    "name": "Stationeer (Vanilla)",
+    "author": "jeremiejt38@gmail.com",
```

Comment:
dont change the Author

Reply:
It's a new Egg, not an edit of the old one. So Pterodactyl Interface automatically add my own email address.

Reply:
> It's a new Egg, not an edit of the old one. So Pterodactyl Interface automatically add my own email address.

its not a.new Egg. you used my Egg, so change Back The author

Reply:
I'm supposed to put the old author of the old egg because this one isn't the same?